Dashboard/app/js/lib/views/maps/map-views-common.js

```diff
@@ -774,6 +774,15 @@ FLOW.NavMapsView = FLOW.View.extend({
                         clickedPointContent += srcAttr + signatureJson.image +'"/></div>';
                         clickedPointContent += '<div class="signedBySection">'+Ember.String.loc('_signed_by') +': '+signatureJson.name+'</div>';
                         break;
+                      case "CADDISFLY":
```

Comment:
I was just wondering if this section isn't doing the same thing as `FLOW.renderCaddisflyAnswer()` in `views.js`. Can you review and if its the case then reuse the function. 👍